### PR TITLE
Adds known issue for early access users when upgrading

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-8145

Link to docs preview:
https://66294--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-known-issues

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1819,6 +1819,14 @@ As a workaround, if you have an installation configuration file, update it with
 
 * When you deploy a cluster with a Network Type of `OVNKubernetes` on {ibmpowerProductName}, compute nodes may reboot because of a kernel stack overflow. As a workaround, you can deploy the cluster with a Network Type of `OpenShiftSDN`. (link:https://issues.redhat.com/browse/RHEL-3901[*RHEL-3901*])
 
+* The following known issue applies to users who upgraded their {product-title} deployment to an early access version of {product-version} using release candidate 3 or 4: 
++
+With the introduction of the node identify feature, some pods are running as root. For users who updated to an early access version of {product-title} {product-version}, attempting to upgrade to the official {product-version} might not progress with the Network Operator reporting the following state: `DaemonSet "/openshift-network-node-identity/network-node-identity" update is rolling`.
```
Comment (jcaamano) [inline release_notes/ocp-4-14-release-notes.adoc:1824]: I would rephrase this:
```
With the introduction of the node identify feature, some pods are running as root. For users who updated ...
```
to 
```
After the introduction of the node identify feature, some pods that were running as root will be updated to run unprivileged. As a consequence, for users who updated ...
```

and
```
not progress with the Network Operator reporting the following state:
```
to
```
not progress with the Network Operator reporting the following state persistently:
```

Just an idea, edit as you see fit

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1819,6 +1819,14 @@ As a workaround, if you have an installation configuration file, update it with
 
 * When you deploy a cluster with a Network Type of `OVNKubernetes` on {ibmpowerProductName}, compute nodes may reboot because of a kernel stack overflow. As a workaround, you can deploy the cluster with a Network Type of `OpenShiftSDN`. (link:https://issues.redhat.com/browse/RHEL-3901[*RHEL-3901*])
 
+* The following known issue applies to users who upgraded their {product-title} deployment to an early access version of {product-version} using release candidate 3 or 4: 
++
+After the introduction of the node identify feature, some pods that were running as root are updated to run unprivileged. For users who updated to an early access version of {product-title} {product-version}, attempting to upgrade to the official {product-version} might not progress with the Network Operator reporting the following state persistently: `DaemonSet "/openshift-network-node-identity/network-node-identity" update is rolling`.
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1824]: >  might not progress with the Network Operator reporting

I'm not clear on whether what _the Network Operating reporting_ is good or bad. Is the update supposed to progress with this state, or does the state indicate something is wrong?

Comment (mburke5678) [reply release_notes/ocp-4-14-release-notes.adoc:1824]: I believe we update cluster and upgrade Operators. 
```suggestion
After the introduction of the node identify feature, some pods that were running as root are updated to run unprivileged. For users who updated to an early access version of {product-title} {product-version}, attempting to update to the official {product-version} might not progress with the Network Operator reporting the following state persistently: `DaemonSet "/openshift-network-node-identity/network-node-identity" update is rolling`. 
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -1819,6 +1819,14 @@ As a workaround, if you have an installation configuration file, update it with
 
 * When you deploy a cluster with a Network Type of `OVNKubernetes` on {ibmpowerProductName}, compute nodes may reboot because of a kernel stack overflow. As a workaround, you can deploy the cluster with a Network Type of `OpenShiftSDN`. (link:https://issues.redhat.com/browse/RHEL-3901[*RHEL-3901*])
 
+* The following known issue applies to users who upgraded their {product-title} deployment to an early access version of {product-version} using release candidate 3 or 4:
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1822]: I believe we update cluster and upgrade Operators. 

```suggestion
* The following known issue applies to users who updated their {product-title} deployment to an early access version of {product-version} using release candidate 3 or 4: 
```

**Comment by mburke5678** [comment]
A coupe of nits. Otherwise LGTM
